fili-core/src/main/java/com/yahoo/bard/webservice/druid/model/response/DruidResponseSerializer.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2016 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.druid.model.response;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+
+import java.io.IOException;
+
+/**
+ * Serializes {@link DruidResponse} into a json equivalent to Druid's actual responses.
+ */
+public class DruidResponseSerializer extends JsonSerializer {
+
+    @Override
+    public void serialize(Object o, JsonGenerator jsonGenerator, SerializerProvider serializerProvider)
+            throws IOException {
```

Comment:
Yea I was just following the same way as `DimensionToNameSerializer` did it.

As far as I can tell this logic never gets passed to callers directly (unless they instantiate a `DruidResponseSerializer`) because they go through the `ObjectMapper` which I'm guessing is handling the exception. 

Let me know if you still want me to catch it because I'm not sure which way is expected with jackson.